src/box.jl

```diff
@@ -31,8 +31,8 @@ end
 
 function normal(x::SVector{N}, b::Box{N}) where {N}
     d = b.p * (x - b.c)
-    (m,i) = findmin(abs.(abs.(d) - b.r))
-    return SVector{N}(b.p[i,:]) * sign(d[i])
+    ~, i = findmin(abs.(abs.(d) - b.r))
+    return normalize(b.p[i,:] * sign(d[i]))  # b.p[i,:] is non-unit for non-rectangular box
```

Comment:
Oops...  I missed your earlier comment on `~`.  I thought it works as in MATLAB.  Sorry that I already merged.

I guess I will need to put another PR to fix this.  Is there a better way to unmerge it?